source/simulator/parameters.cc

```diff
@@ -1705,6 +1731,18 @@ namespace aspect
     }
     prm.leave_subsection ();
 
+    prm.enter_subsection ("Volume of Fluid");
+    {
+      const bool explicit_volume_of_fluid_tracking_enabled = prm.get_bool("Enable interface tracking");
```

Comment:
I believe that this may have been left over from when this option indicated that it was necessary to load the VoF handler which then did all the parsing. I have checked, and since it does appear to be redundant, I have removed the parameter.